lockbox-ios/View/ItemDetailView.swift

```diff
@@ -194,7 +194,8 @@ extension ItemDetailView: UIGestureRecognizerDelegate {
                 }
                 
                 if let revealObserver = cellConfiguration.revealPasswordObserver {
-                    cell.textValue.font = UIFont(name: "Menlo-Regular", size: 16)
+                    // Font is not consistent with app. Uncomment this line out if this was intentional 
+//                    cell.textValue.font = UIFont(name: "Menlo-Regular", size: 16)
```

Comment:
I think these lines can just be deleted 👍

Reply:
Done!

Reply:
@eliserichards For some reason I keep getting "build error" for the uispecs test in GitHub. Is that error on my side?

Reply:
@KrisJackson  no it isn't - figuring it out now. It's an error with our python script that creates fake logins for the main list, I believe